docs/encyclopedia/nexus-operations.mdx

```diff
@@ -0,0 +1,184 @@
+   2. Retries are processed by the Nexus Machinery in the caller’s Temporal cluster and tracked as part of the caller Workflow, similar to Activities.
+3. Nexus Operation handler processes the Nexus Operation.
+   1. Handler Worker gets a Nexus Task from Task Queue.
+   2. Operation handler starts a Workflow in the target cluster with attached Nexus Callback.
+   3. Nexus Operation ID is returned since the Nexus Operation is asynchronous.
+   4. `NexusOperationStarted` is added to caller’s Event History.
+   5. Operation ID is associated with the caller's `NexusOperationStarted` event and the caller Workflow is unblocked if waiting for `NexusOperationStarted` using the Temporal SDK.
+4. Handler’s Nexus Machinery sends a Nexus completion Callback when the Workflow reaches a terminal state.
+   1. Operation result is returned via callback since the operation is asynchronous.
+   2. `NexusOperationCompleted` is added to caller’s Event History.
+5. Caller’s Workflow code is returned the result if requested.
+
+### Synchronous Operation lifecycle
+
+A synchronous Nexus Operation handler has less than 10 seconds to complete, as measured from the caller's Nexus Machinery, and will then timeout with a retryable error.
+The handler should stay within the context deadline to not timeout.
+The Nexus Operation will be retried by the caller's Nexus Machinery, until the operation's Schedule-to-Close timeout has been exceeded.
+
+The lifecycle of a synchronous Nexus Operation, for example to do a Query or a Signal:
+
+1. Caller’s Workflow executes a Nexus Operation using the Temporal SDK.
+   1. **`NexusOperationScheduled`** added to caller’s Event History.
+   2. Nexus Operation added to Nexus Machinery in the caller’s Temporal cluster.
+   3. Caller Workflow doesn’t automatically block on a Nexus Operation.
+   4. Caller’s Workflow can interact with the Nexus Operation execution:
+      1. Wait for the Nexus Operation to start (not needed for synchronous operations).
+      2. Wait for the Operation result.
+      3. Cancel the Nexus Operation (not allowed for synchronous operations).
+2. The caller’s Temporal cluster attempts to start the Nexus Operation on the target cluster via the Nexus Endpoint.
+   1. Nexus Machinery in the target cluster sync matches the Nexus request to a Worker polling the endpoint's target Namespace and Task Queue.
+   2. Retries are processed by the Nexus Machinery in the caller’s Temporal cluster and tracked as part of the caller Workflow, similar to Activities.
+3. Nexus Operation handler processes the Nexus Operation.
+   1. Handler Worker gets a Nexus Task from Task Queue.
+   2. Operation handler is executed, which may use the Temporal client to Signal or Query a workflow.
+   3. Operation result is returned inline, since the operation is synchronous.
+   4. **`NexusOperationCompleted`** added to caller’s Event History.
+4. Caller’s Workflow code is returned the result if requested.
```

Comment:
nit: you can use `1.` for all of the bullets and markdown will automatically add the right numbers in for you. It's easier to refactor later this way.

Reply:
fixed